.github/workflows/build-deploy-app-v2.yaml

```diff
@@ -0,0 +1,151 @@
+name: Github Environment - DPIA Web App v2
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "src/frontend/**"
+  workflow_dispatch:
+  
+jobs:
+  DPIA-App-Build-Tag-Push:
+    runs-on: ubuntu-latest
+    environment: APP-Dev
+    steps:
+    
+      # Checkout CIRMO-DPIA Github Repository
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      
+      # Login the BC Gov Openshift (DPIA TOOLS Namespace)
+      - name: Cluster Login -> TOOLS Namespace
+        uses: redhat-developer/openshift-actions@v1.1
+        with:
+          openshift_server_url: ${{ secrets.OPENSHIFT4_SERVER_URL }}
+          parameters: '{"apitoken": "${{ secrets.OPENSHIFT4_TOOLS_SA_PASSWORD }}"}'
+          cmd: |
+            'version'
+            
+      # Login to BC Gov Docker Image Repository
+      - name: Login to Openshift Docker
+        run : |
+          docker login ${{ secrets.OPENSHIFT4_PUBLIC_IMAGE_REPOSITORY_BASE_URL }} -u ${{ secrets.OPENSHIFT4_TOOLS_SA_USERNAME }} -p ${{ secrets.OPENSHIFT4_TOOLS_SA_PASSWORD }}
+      
+      # Build the DPIA Frontend Application Image via docker-compose
+      - name: Build the Docker image
+        run: |
+          sudo docker-compose build dpia-webapp
+      
+      # TROUBLESHOOTING - view images present in github action run
+      - name: Docker List Images
+        run: |
+          docker image ls 
+      
+      # Tag Docker Image built with latest commit hash (ex// $HASH-dev)
+      - name: Docker Tag Version
+        run: |
+          docker tag cirmo-dpia_dpia-webapp ${{ secrets.OPENSHIFT4_APP_PUBLIC_IMAGE_REPO_DEV }}:latest
+      
+      # TROUBLESHOOTING - view images present in github action run
+      - name: Docker List Images
+        run: |
+          docker image ls 
+          
+      # Push docker image to BC Gov Repository
+      - name: Docker Push Version
+        run: |
+          docker push ${{ secrets.OPENSHIFT4_APP_PUBLIC_IMAGE_REPOSITORY }}:latest
```

Comment:
Is this right? I thought we had a different repo/imagestream per env now

Reply:
yes, I forgot to change this. should be reflected in my last commit.